purchase_work_acceptance_late_fine/models/account_move.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+    _description = "Journal Entries"
+
+    late_wa_id = fields.Many2one(
+        comodel_name="work.acceptance",
+        string="Late WA",
+        domain=lambda self: self._get_fine_amount(),
+    )
+
+    def _get_fine_amount(self):
+        wa = self.env["work.acceptance"].search([("late_fine", ">", 0)])
+        return [("id", "in", wa.ids)]
+
+    @api.onchange("late_wa_id")
+    def _onchange_wa_id(self):
+        self.line_ids = False
+        # Copy partner.
+        self.partner_id = self.late_wa_id.partner_id
+        wa_lines = self.late_wa_id
```

Comment:
Just 1 WA, no (s). Also, we shouldn't need to use for...loop, right? Because, we don't have multiple WA here.